mint-signup-form.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://kit.fontawesome.com/774f0ae2db.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" type="text/css" href="main.css">
+        <title>Mint Sign Up Page</title>
+    </head>
+    <body>
+        
+        <header class="page-header">            
+            <div>
+                <p>Already have an Intuit account?</p>
+                <p><a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS#"> forgot my User ID or Password </a></p>
+            </div>
+            <div>
+                <button id="sign-in-btn" class="btn" type="submit"  value="Signin"><i class="fa fa-lock"></i> Sign in</button>
+                    
+            </div>            
+        </header>
+        
+        <main>
+            <div id="logos">
+                <div id="site-logo">
```

Comment:
# [REQUIRED] - LOGO SECTION
## The logo seems not to be in its right place.
- It should be rightly placed in the top right corner. Kindly see to that. It is a requirement.

![sign_in_not_rightly_placed](https://user-images.githubusercontent.com/22345961/78132908-3087b200-7426-11ea-86d4-4564df43a351.png)

## Styling
remove the underline styling. You can do this with CSS to match the original clone.

From this here:
![i_forgot_password](https://user-images.githubusercontent.com/22345961/78133159-9ecc7480-7426-11ea-8782-f735550bc566.png)

To this here:
![i_forgot_password_original](https://user-images.githubusercontent.com/22345961/78133198-b0158100-7426-11ea-80e9-b71392ac9dac.png)

PS: Also add the 'I'. You seem to have forgotten about it 😉

Reply:
Done